tests/Microsoft.Bot.Builder.ApplicationInsights.Tests/TelemetryWaterfallTests.cs

```diff
@@ -120,16 +126,30 @@ public async Task EnsureEndDialogCalled()
             var telemetryClient = new Mock<IBotTelemetryClient>();
             var saved_properties = new Dictionary<string, IDictionary<string, string>>();
             var counter = 0;
+
             // Set up the client to save all logged property names and associated properties (in "saved_properties").
             telemetryClient.Setup(c => c.TrackEvent(It.IsAny<string>(), It.IsAny<IDictionary<string, string>>(), It.IsAny<IDictionary<string, double>>()))
                             .Callback<string, IDictionary<string, string>, IDictionary<string, double>>((name, properties, metrics) => saved_properties.Add($"{name}_{counter++}", properties))
                             .Verifiable();
-            var waterfallDialog = new MyWaterfallDialog("test", new WaterfallStep[]
+            var steps = new WaterfallStep[]
```

Comment:
Can this be replaced by the method NewWaterfall()? I think this will reduce duplicated code